dockerfiles/SparkScalaContainerDockerfile

```diff
@@ -1,3 +1,5 @@
+# This ARG isn't used but prevents warnings in the build script
```

Comment:
Should we also add this note to `RPCDockerfile` and `ClipperLibBaseDockerfile` ?

Reply:
Done.